yt_dlp/downloader/external.py

```diff
@@ -272,6 +280,9 @@ def _make_cmd(self, tmpfilename, info_dict):
         cmd += self._bool_option('--show-console-readout', 'noprogress', 'false', 'true', '=')
         cmd += self._configuration_args()
 
+        if info_dict.get('__rpc_port'):
+            cmd += ['--enable-rpc', f'--rpc-listen-port={info_dict["__rpc_port"]}']
```

Comment:
would it be good to enable auth here? (`--rpc-secret` maybe?)

Reply:
I think so